cmd/lora-gateway-bridge/cmd/configfile.go

```diff
@@ -129,6 +129,9 @@ tls_cert="{{ .Backend.MQTT.TLSCert }}"
 
 # mqtt TLS key file (optional)
 tls_key="{{ .Backend.MQTT.TLSKey }}"
+
+# maximum seconds that will be waited between reconnection attempts when connection is lost
+max_reconnect_interval_seconds="{{ .Backend.MQTT.MaxReconnectIntervalSeconds }}"
```

Comment:
Could you rename this to `max_reconnect_interval` and `MaxReconnectIntervalSeconds` to `MaxReconnectInterval`?

E.g.

```
# Max reconnect interval.
#
# The maximum interval that will be waited between reconnection attempts when the connection is lost.
+max_reconnect_interval="{{ .Backend.MQTT.MaxReconnectInterval }}"
```